PRIVACY.md

```diff
@@ -0,0 +1,47 @@
+## Privacy Notice
+
+This privacy notice discloses the privacy practices for Flagrow Bazaar. This privacy notice applies solely to information collected by this application and its associated website at flagrow.io. It will notify of the following:
+
+* What personally identifiable information is collected from YOU through the website, how it is used and with whom it may be shared.
```

Comment:
Regarding MD formatting, the spacing after third-level titles is inconsistent :sweat_smile: